src/components/Menu.jsx

```diff
@@ -1,29 +1,75 @@
 import React, { useState, useEffect } from "react";
-import { Segment, List } from "semantic-ui-react";
+import { Segment, Grid, Menu, Container } from "semantic-ui-react";
 import MenuItem from "./MenuItem";
 import axios from "axios";
 
-const Menu = () => {
+const MenuPage = () => {
   const [menuItems, setMenuItems] = useState([]);
+  const [activeCategory, setActiveCategory] = useState("starters");
+
   useEffect(() => {
     axios.get("https://slowfood.heroku.com/api/products").then((response) => {
       setMenuItems(response.data.products);
     });
   }, []);
+  useEffect(() => {
+    axios.get("https://slowfood.heroku.com/api/products").then((response) => {
+      setMenuItems(response.data.products);
+    });
+  }, [activeCategory]);
+
+  const filteredCategory = menuItems.filter(
+    (item) => item.category === activeCategory
+  );
 
-  let menuList = menuItems.map((item) => {
-    return (
-      <List data-cy={`item-${item.id}`}>
-        <MenuItem item={item} />
-      </List>
-    );
+  let menuList = filteredCategory.map((item) => {
```

Comment:
also about naming. It is best to follow this standard `plural.map((singular) => {...})` example `items.map((item) => {...})`. Here you have `category` and `item` doesn't make sense.